frontend/simple/views/containers/contributions/MessageMissingIncome.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="box is-unfilled c-box" v-on="$listeners">
```

Comment:
Why is `v-on="$listeners"` here? It's not clear to me this is [how the docs intended](https://vuejs.org/v2/api/#vm-listeners) it to be used? (Same question for the other place `v-on="$listeners"` is used in another file)

Reply:
It' exactly that docs say, you can add `@click="handleFormTriggerClick"` to the component when using it only by using `v-on="$listeners"`, without it it `@click` (or any other event) would be ignored...